### DISCUSSION TITLE
Offline mode

### DISCUSSION THREAD

**Discussion by btk-42** [firefly-iii/firefly-iii#6134] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [ ] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [ ] ... there are no open or closed issues that are related to my problem
- [ ] ... it's [definitely me, not you](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

Hello, 

I have sometimes offline mode on a laptop and almost every time on mobile phone. I'm using Firefox on the laptop and safari on iPhone. 
To solve that problem I have to open my firefly in incognito mode and then I'm able to open webpage.  Clearing cache is working also but only on the laptop. 

### Expected behaviour

I would like to open firefly.

### Debug information

Probably everything is in description.

**Comment by JC5** [comment]
It's a cache issue like you say, there's not much I can do about it unfortunately.

**Reply by btk-42** [reply]
Okay, anyway. Thank you

**Comment by Bazinga88** [comment]
I would like to add the folowing because i had the same issue multiple times last month. 
BTW, the offline page has a empty square in it where a picture belongs probably.

Running ubuntu server 20.04, and the requirements givin in the installation manual.
Connecting with a iphone 13

**Scenario:**
- Connect via home wifi to a local domain 'https://finance.whatever.home' that points to firefly-iii.
- Success. you can reconnect again and again on the same local wifi connection, even after a iphone reboot. 
- Go offsite. 
- Connect via OVPN to home.
- Go to your local domain 'https://finance.whatever.home' that points to firefly-iii. 
- Offline page is there. You will need to clear cache. In safari it will succeed. Chrome does not understand. 
  Remove app Chrome, clear cache and then open in safari is success. Reinstall chrome then it wil also work. 
  But connected over wifi at home it will fail again if you did not do the previous.
- Back at home. Offline page is there over the initially wifi connection. 

I just updated to the latest FF-iii version and still the same. 
Indeed you have to clear the cache but this interupts also everything else. 
Is there a workaround or solution for this?

I think something goes wrong if the path to the webserver changes. 
Because you use laravel, is this a common issue? Just asking :-)

Thanks for reading.

**Comment by JC5** [comment]
The offline mode will be gone in future versions of Firefly III. So this will eventually no longer be an issue.
